modules/nodes-cluster-resource-levels-command.adoc

```diff
@@ -44,44 +44,51 @@ spec:
 
 .Procedure
 
-To use the tool on the command line: 
+To use the tool on the command line:
 
-. Run the following command:
+. From the terminal, run:
++
+----
+podman login registry.redhat.io
+----
++
```

Comment:
Might not be hurting anything, but this "+" continuation character isn't necessary here since the previous line is the end of the step

Reply:
Done